src/ExtranetBundle/objects/db/TDataExtranetUser.class.php

```diff
@@ -936,6 +936,20 @@ public function __sleep()
         return array('table', 'oBillingAddress', 'oShippingAddress', 'aSessionData', 'sqlData', 'pageAccessCache');
     }
 
+    // isRequestForResource will try to identify the request based on its path.
+    // This method is quite specific and only used when deciding if it should check the active portal of the user
+    // on session validation. See https://github.com/chameleon-system/chameleon-system/issues/787 for details.
+    private function isRequestForResource(): bool {
+        $pathInfo = $this->getCurrentRequest()->getPathInfo();
+
+        $isResourceRequest =
+            str_starts_with($pathInfo, '/_wdt') ||
+            str_ends_with($pathInfo, '.css') ||
+            str_ends_with($pathInfo, '.jpg');
```

Comment:
I would suggest to set paths instead: /chameleon/mediapool, /chameleon/outbox, /_wdt.
Otherwise you would need a full list of file extensions. Like .png, .webp, .pdf and other.

Reply:
There might be other (future, custom) resources with other suffixes or paths (*.js for example)?

Reply:
The path is a good idea. I will test that. But for the other file formats: as far as I know none of them would trigger a kernel request as we are not handling those formats internally.

Reply:
gif is also a possible request (missing image).

Is the pathInfo always lower case?